sql/core/src/test/resources/sql-tests/inputs/sql-session-variables.sql

```diff
@@ -26,9 +26,9 @@ DECLARE VARIABLE IF NOT EXISTS var1 INT;
 DROP TEMPORARY VARIABLE IF EXISTS var1;
 
 SET VARIABLE title = 'Drop Variable';
-DECLARE VARIABLE var1 INT;
+DECLARE VAR var1 INT;
```

Comment:
Update only `one SQ`L in `sql-session-variables.sql` so that UT can cover it.

Reply:
do we have a parser suite for variable? I think that's a better place.

Reply:
if not maybe we should create one, like `ShowTablesParserSuite`

Reply:
- Currently we don't have a `parser suite` for variable, let me create one. 
- Additionally, there are already `sql-session-variables.sql` assisted tests available. Do we still need to create some UT like `DeclareVariableSuiteBase`, `v1\DeclareVariableSuite`, `v2\DeclareVariableSuite` ... ? 
(I'm not sure, it looks a bit `duplicate`, does it?)

Reply:
We only support session variable today so we don't need cross-test with v1/v2.

Reply:
Okay